collections/app/controllers/CollectionsController.scala

```diff
@@ -157,6 +158,7 @@ class CollectionsController(authenticated: Authentication, config: CollectionsCo
           s"$collectionPath has children, can't delete!"
         )
       } else {
+        logger.info(s"Deleting collection ${path.mkString("/")}")
```

Comment:
Do we have any user information we can add here?

Reply:
yes, had to have a good look for the right invokation